content/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages.md

```diff
@@ -136,7 +136,7 @@ To ensure the repository's URL is correct, replace REPOSITORY with the name of t
 
 You can install packages from {% data variables.product.prodname_registry %} by adding the packages as dependencies in the *package.json* file for your project. For more information on using a *package.json* in your project, see "[Working with package.json](https://docs.npmjs.com/getting-started/using-a-package.json)" in the npm documentation.
 
-By default, you can add packages from one organization. For more information, see [Installing packages from other organizations](#installing-packages-from-other-organizations)
+By default, you can add packages from one organization. For more information, see "[Installing packages from other organizations](#installing-packages-from-other-organizations)".
```

Comment:
This is super counterintuitive, and I'll admit that I actually prefer the British usage, but the GitHub docs use the American usage of the period and it needs to go inside the quotes here. Here's a reference [Blog](https://www.grammarly.com/blog/period/) from Grammarly that explains it a bit. 

```suggestion
By default, you can add packages from one organization. For more information, see "[Installing packages from other organizations](#installing-packages-from-other-organizations)."
```

Reply:
Perfect! I agree. Just noticed the use of quotes in the rest of the document and my OCD kicked in 😆 
Thanks!